packages/ngircd/fix-hardcoded-path.patch

```diff
@@ -0,0 +1,64 @@
+--- ngircd-26/autogen.sh	2021-11-01 11:27:20.660709063 +0700
++++ ngircd-26.mod/autogen.sh	2021-11-01 11:54:22.496709160 +0700
+@@ -68,7 +68,7 @@
+ 	major="$2"
+ 	minor=99
+ 
+-	[ -n "$PREFIX" ] && searchlist="${PREFIX}/$1 ${PREFIX}/bin/$1 $searchlist"
++	[ -n "$TERMUX_PREFIX" ] && searchlist="${TERMUX_PREFIX}/$1 ${TERMUX_PREFIX}/bin/$1 $searchlist"
```

Comment:
Use @TERMUX_PREFIX@ in patches, but as mentioned above I don't think you need to change anything in this script